### ISSUE TITLE
RSS feed for the blog

### ISSUE THREAD

**Issue by dannyob** [EFForg/sec#351]
We don't necessarily need a UI for this, just a <link rel="alternate" type="application/rss+xml" title="RSS"
      href="https://sec.eff.org/blog/rss"> or somesuch would do it.

**Comment by lschatzkin** [comment]
@wioux @vbrown608 So I thought I'd take a stab at this but am getting a little lost. Since we are creating the blog from a feed, is there a direct way to just use that feed?  Or do I need to create a feed of the blog posts such as suggested here? https://www.codingfish.com/blog/129-how-to-create-rss-feed-rails-4-3-steps

**Comment by wioux** [comment]
Yeah I think it would be best to link directly to the same feed SEC is using itself: https://www.eff.org/deeplinks.xml?field_issue_tid=11461

**Comment by lschatzkin** [comment]
Allright, I'll give it another shot. I think I was overcomplicating things.
